monocle/github/pullrequest.py

```diff
@@ -257,6 +258,10 @@ def extract_pr_objects(pr):
                 change['repository_fullname'],
                 change['number'],
             )
+            change['url'] = ('%s/%s/pull/%s'
```

Comment:
You need to add "url" in elastic schema in db.py.

Reply:
See this change: https://github.com/morucci/monocle/pull/28

Reply:
ok